app/lib/account/consent_backend.dart

```diff
@@ -263,10 +261,8 @@ class ConsentBackend {
       () async {
         await action?.onDelete(consent);
         await withRetryTransaction(_db, (tx) async {
-          final c =
-              await tx.lookupValue<Consent>(consent.key, orElse: () => null);
-          if (c == null) return;
-          tx.delete(c.key);
+          final c = await tx.lookupOrNull<Consent>(consent.key);
+          if (c != null) tx.delete(c.key);
```

Comment:
Why not simply `tx.delete(consent.key)`? I assume we would have to handle the case where it's already deleted.. but that's about it?

Reply:
This is the consequence of the same outer retry block. This should be removed once the onaccept and the transaction is merged.